### PR TITLE
fix(docs): Language selector showing `EN` as active language on `non-EN` pages

### PR DESCRIPTION
**Pull request by AyanavaKarmakar**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

The active language selector was hard coded before to `EN`. Updated the code so that the active option syncs with the current language.

## Screenshots

- Before

image

- After

image

💯

### CODE REVIEW

File: www/src/components/navigation/moreMenu.astro [hunk 1]
```diff
@@ -45,7 +47,43 @@ const isRtl = getIsRtlFromUrl(pathname);
             ></path>
           </svg>
         </span>
-        <span>Edit this page</span>
+        <span> Edit this page</span>
+      </a>
+    </li>
+
+    <li
+      class={clsx(
```
Comment (c-ehrlich) [inline www/src/components/navigation/moreMenu.astro:55]: i think this belongs in a different PR?

Comment (AyanavaKarmakar) [reply www/src/components/navigation/moreMenu.astro:55]: oops

**Review by c-ehrlich — APPROVED** [review]
Fix looks good. Thanks :)
